content/concepts/the-rippled-server/rippled-server-modes.md

```diff
@@ -2,61 +2,83 @@
 
 The `rippled` server software can run in several modes depending on its configuration, including:
 
-* Stock server - follows the network with a local copy of the ledger.
-* Validating server, or _validator_ for short - participates in consensus (and does everything a stock server does, too).
-* `rippled` server in stand-alone mode - for testing. Does not communicate to other `rippled` servers.
+- [**Network Mode**](#network-mode) - This is the main mode of the server: it follows the peer-to-peer network, processes transactions, and maintains some amount of [ledger history](ledger-history.html). This mode can be configured to do any or all of the following roles:
+    - [**Validator**](#validators) - Helps secure the network by participating in consensus
+    - [**API Server**](#api-servers) - Provides [API access](get-started-with-the-rippled-api.html) to read data from the shared ledger, submit transactions, and watch activity in the ledger. Optionally, this can be a [**Full History Server**](#full-history-servers), which keeps a complete record of transaction and ledger history.
```

Comment:
Isn't a Reporting Mode server also an API Server? I find that having API Server as a subcategory of Network Mode confusing.

Reply:
I struggled with this too. Hard to divide up the modes when there's so much mix-and-match overlap. The point here is that this is something Network Mode (P2P Mode?) _can_ do. Not sure if there's a way to concisely do that while also pointing out that this is what Reporting Mode does.

If something has to be sacrificed here, I think it makes sense to emphasize that being an API server is something Network Mode does/can do, because Reporting Mode is really just a very specialized interface for doing the same thing at scale; it's really much easier not to think much about Reporting Mode until you're encountering scaling problems with your P2P server.